### PR TITLE
Add scripts to compare performance between two branches

### PR DESCRIPTION
**Pull request by blikblum**
Add scripts that compares performance of the current branch checked in the local repository and a base one (develop).

Steps:
 * builds current branch
 * checkout develop
 * builds develop branch
 * checkout current branch
 * run perf suites based on benchmark.js

Add two npm scripts: 
  * perf:all -> build the two branches and run perf suite
  * perf -> run perf suite

This PR contains changes from #27 

Here is the output of perf suite comparing #27 and develop:

Benchmarking no props
hyperscript x 13,230,387 ops/sec ±5.07% (91 runs sampled)
jsx-base x 699,560 ops/sec ±2.24% (98 runs sampled)
jsx-new x 676,120 ops/sec ±2.98% (93 runs sampled)
no props - fastest is hyperscript   fastest JSX is  [ 'jsx-base' ]

Benchmarking simple props
hyperscript x 13,216,166 ops/sec ±0.73% (98 runs sampled)
jsx-base x 41,596 ops/sec ±0.76% (92 runs sampled)
jsx-new x 45,047 ops/sec ±0.57% (98 runs sampled)
simple props - fastest is hyperscript   fastest JSX is  [ 'jsx-new' ]

Benchmarking module object props
hyperscript x 9,655,921 ops/sec ±0.62% (101 runs sampled)
jsx-base x 41,222 ops/sec ±2.70% (97 runs sampled)
jsx-new x 34,262 ops/sec ±4.33% (82 runs sampled)
module object props - fastest is hyperscript   fastest JSX is  [ 'jsx-base' ]

Benchmarking modules attribute props
hyperscript x 7,928,043 ops/sec ±5.80% (81 runs sampled)
jsx-base x 28,325 ops/sec ±2.04% (89 runs sampled)
jsx-new x 29,938 ops/sec ±1.11% (102 runs sampled)
modules attribute props - fastest is hyperscript   fastest JSX is  [ 'jsx-new' ]

Benchmarking real world form
jsx-base x 2,844 ops/sec ±1.19% (99 runs sampled)
jsx-new x 3,024 ops/sec ±0.66% (101 runs sampled)
real world form - fastest is jsx-new   fastest JSX is  [ 'jsx-new' ]

All benchmarks complete

### CODE REVIEW

**Comment by blikblum** [comment]
For the record this is the results of my changes, that were published as [snabbdom-pragma-lite](https://www.npmjs.com/package/snabbdom-pragma-lite):

Benchmarking no props
hyperscript x 12,807,054 ops/sec ±2.19% (88 runs sampled)
jsx-base x 740,841 ops/sec ±0.40% (104 runs sampled)
jsx-new x 3,146,969 ops/sec ±0.56% (106 runs sampled)
no props - fastest is hyperscript   fastest JSX is  [ 'jsx-new' ]

Benchmarking simple props
hyperscript x 12,201,875 ops/sec ±0.56% (107 runs sampled)
jsx-base x 42,757 ops/sec ±0.69% (107 runs sampled)
jsx-new x 1,201,914 ops/sec ±0.47% (105 runs sampled)
simple props - fastest is hyperscript   fastest JSX is  [ 'jsx-new' ]

Benchmarking module object props
hyperscript x 9,248,187 ops/sec ±0.64% (107 runs sampled)
jsx-base x 43,626 ops/sec ±0.56% (107 runs sampled)
jsx-new x 596,333 ops/sec ±0.42% (101 runs sampled)
module object props - fastest is hyperscript   fastest JSX is  [ 'jsx-new' ]

Benchmarking modules attribute props
hyperscript x 9,070,325 ops/sec ±0.51% (109 runs sampled)
jsx-base x 30,560 ops/sec ±0.70% (107 runs sampled)
jsx-new x 320,799 ops/sec ±0.52% (102 runs sampled)
modules attribute props - fastest is hyperscript   fastest JSX is  [ 'jsx-new' ]

Benchmarking real world form
jsx-base x 2,921 ops/sec ±1.42% (103 runs sampled)
jsx-new x 34,647 ops/sec ±0.61% (102 runs sampled)
real world form - fastest is jsx-new   fastest JSX is  [ 'jsx-new' ]

**Comment by blikblum** [comment]
Travis is passing now

File: .travis.yml [hunk 1]
```diff
@@ -11,9 +11,7 @@ matrix:
 script:
   - yarn run lint
   - yarn run test:prebuild
-  - export HASH=$(cksum dist/index.js)
   - yarn run build
-  - bash -c 'if [[ "$(cksum dist/index.js)" != $HASH ]]; then exit 1; fi'
```
Comment (Swizz) [inline .travis.yml]: This test is here to ensure I build the lib before pushing.
Why do you need to remove it ?

Comment (blikblum) [reply .travis.yml]: In other projects i contribute, its requested to not not commit the updated build file in PR. It does only in release.

If you want i can commit the build file in the PR and leave this check

Comment (Swizz) [reply .travis.yml]: idk I took this habbit here, what is better ?

**Comment by blikblum** [comment]
Not commiting the build at each PR generates smaller diffs making easier to see what really changed. 

This is the general rule in JS projects

**Comment by Swizz** [comment]
I want to request the build on a PR on master at least.

Master should be only a release using git flow
https://danielkummer.github.io/git-flow-cheatsheet/

**Comment by blikblum** [comment]
This PR is on develop. 
Should i commit the build files?
Revert the travis changes?

Any thing is good to me

**Comment by Swizz** [comment]
Go for Travis revert + build. 
I prefer to ensure the lib to be built on PR merge

**Comment by blikblum** [comment]
Done

**Comment by Swizz** [comment]
Npm published with [v2.7.0](https://github.com/Swizz/snabbdom-pragma/releases/tag/2.7.0)

**Pull request merged**